README.md

```diff
@@ -1,64 +1,86 @@
+##### MAAS Deployment Components
 
-The following instructions assume that nodes `infra1`, `infra2`, `infra3` are deployed with Ubuntu 22.04 LTS and their networking is properly configured.
+MAAS Anvil is part of the MAAS deployment strategy, which includes:
 
-In addition, the instructions assume that MAAS Anvil will deploy all the available components (roles) in all three nodes:
+1. **MAAS Charms**: [Charmed](https://juju.is/docs/juju/charmed-operator) versions of MAAS components.
+    - [MAAS Agent Charm](https://charmhub.io/maas-agent)
+    - [MAAS Region Charm](https://charmhub.io/maas-region)
+2. [**MAAS Anvil**](https://github.com/canonical/maas-anvil): Uses MAAS charms to simplify MAAS deployments.
+3. [**MAAS Terraform Provider**](https://registry.terraform.io/providers/maas/maas/latest/docs): Configures active MAAS deployments.
 
-- MAAS region controller
-- MAAS rack controller (agent)
-- PostgreSQL
-- HAProxy
+MAAS Anvil streamlines the deployment process using MAAS charms. After deployment, the MAAS Terraform provider can be used for further configuration of the active MAAS environment.
+
+# In this documentation
+
+| Serve our study                                                      | Serve our work                                                       |
+| :------------------------------------------------------------------- | :------------------------------------------------------------------- |
+| [Tutorials](#tutorial) Hands-on introductions to MAAS Anvil features | [How-to guides](#how-to) Step-by-step guides covering key operations |
+|                                                                      | [Reference](#reference) Technical specifications                     |
+
+# Tutorial
+
+## Bootstrap a maas-anvil cluster to learn the basics
+
+The following instructions assume that you have three nodes `infra1`, `infra2`, `infra3` running Ubuntu 22.04 LTS and their networking is configured correctly.
+
+In addition, the instructions assume that MAAS Anvil deploys all available components (roles) on all three nodes:
+
+-   MAAS region controller
+-   MAAS rack controller (agent)
+-   PostgreSQL
+-   HAProxy
 
 ### Preparation steps for each node
 
+First, MAAS Anvil needs to be installed and some prerequisites for MAAS Anvil need to be set up. This needs to be done on every node. You can learn more about what `maas-anvil prepare-node-script` does in the [CLI interface reference](#maas-anvil-prepare-node-script-options).
+
 ```bash
 ubuntu@infra{1,2,3}:~$ sudo snap install maas-anvil --edge
 ubuntu@infra{1,2,3}:~$ maas-anvil prepare-node-script | bash -x
+```
+
+Among other things, the prepare-node-script adds the current user to the `snap_daemon` group. In order for the group changes to take effect, you must `log out` and `log in` again. If the file ownership of groups is not a major concern for you, you can also run the following command to activate the changes to the groups immediately.
```

Comment:
don't think "log out" and "log in" should be in backticks

Reply:
Good point, code highlighting is probably not appropriate here. I still wanted it to stand out a bit so I am using italics now instead!